am-lib/docs/importer.md

```diff
@@ -0,0 +1,104 @@
+
+For example:
+```public void addRole('citizen', RoleType.IDAM, SecurityClassification.PUBLIC, AccessManagementType.EXPLICIT_ACCESS)```
+
+Possible options for `RoleType`:
+- IDAM
+It's a global (IdAM) role, such as 'citizen', 'caseworker', 'solicitor'.
+
+- RESOURCE
+It's a role related to a resource (for CCD it's "case role"), eg. 'defendant', 'claimant', 'applicant'.
+
+Possible options for `AccessManagementType`:
+- EXPLICIT_ACCESS
+User who has role of this access management type must be granted explicit access if you want him to have any access to a resource. There must be a link (relationship) between him and a resource.
+
+- ROLE_BASED
+User who has role with `ROLE_BASED` access doesn't have to be granted explocit access to a case and he may still have access to it if default permissions for his role (with ROLE_BBASED access) gives him it.
+
+3. Add resource definitions
+By "resource definition" we understand 3 values: service, resource type and resource name (all mandatory, combined unique):
+```public void addResourceDefinition(@NotBlank String serviceName, @NotBlank String resourceType, @NotBlank String resourceName)```
+
+For example:
+```public void addResourceDefinition('CMC', 'CASE', 'CivilMoneyCase')```
+
+Why 'CASE' needs to be provided? At the moment we will support only cases, but in the upcoming releases we are going to support also documents and maybe even other types of resources.
+
+In CCD:
+- serviceName = jurisdiction
+- resourceName = caseType
+- resourceType is always CASE
+
+4. Grant default permissions for a role
+```public void grantDefaultPermission(@NotNull @Valid DefaultPermissionGrant accessGrant```
+
+##How to delete permissions?
+
+You can only truncate all default permissions, in two ways:
+
+1. by `serviceName` and `resourceType`, eg:
+
+```truncateDefaultPermissionsForService("CMC", "CASE");```
+
+It will delete all default permissions for all resource names within provided service.
+
+2. by `serviceName`, `resourceType` and `resourceName` eg:
+
+```truncateDefaultPermissionsForService("CMC", "CASE", "CivilMonayCase");```
+
+will delete all "CivilMonayCase", but no other cases definitions within CMC. One service may have more than 1 case type.
+
+To clean up database you need to delete rows from database in the correct order.
+
+1. Delete default permissions (as described above)
+2. Delete resource definitions
+```public void deleteResourceDefinition(@NotBlank String serviceName, @NotBlank String resourceType, @NotBlank String resourceName)```
+
+for example:
+```deleteResourceDefinition("CMC", "CASE", "CivilMoneyCase")```
+
+3. Delete roles
+```public void deleteRole(@NotBlank String roleName)```
+
+eg:
+```public void deleteRole("citizen")```
+
+This can fail if this role is used as a relationship in `access_management` table.
+
+4. Delete services
+```public void deleteService(@NotBlank String serviceName)```
+
+eg
+```deleteService("CMC")```
+
+No cascade on delete
```

Comment:
Can we remove that or make it headline otherwise paragraph below just repeats what has been already said.